app/models/submission.ts

```diff
@@ -1,10 +1,28 @@
 /* eslint-disable ember/no-computed-properties-in-native-classes, ember/no-mixins */
 import { AsyncBelongsTo, attr, belongsTo } from '@ember-data/model';
 import { ModelBaseMixin } from 'irene/mixins/base-model';
-import { computed } from '@ember/object';
+import { isEmpty } from '@ember/utils';
 
+import ENUMS from 'irene/enums';
 import UserModel from './user';
 
+export interface SubmissionAppData {
+  name: string;
+  package_name: string;
+  icon_url: string;
+  platform: number;
+}
+
+export const statusProgressPercetage = {
```

Comment:
If this would be possibly used elsewhere, it would need a better name `submissionStatusProgressPercentage`. To prevent ambiguity.